### DISCUSSION TITLE
Failed to use ESAPI 2.1.0.1

### DISCUSSION THREAD

**Discussion by AnsticeLi** [ESAPI/esapi-java-legacy#774] [Q&A]
I posted a question on stackoverflow.
hope it is ok to post here, too.

[Failed to use ESAPI 2.1.0.1: AccessController class (org.owasp.esapi.reference.DefaultAccessController) CTOR threw exception]
https://stackoverflow.com/questions/75344918/failed-to-use-esapi-2-1-0-1-accesscontroller-class-org-owasp-esapi-reference-d

**Comment by kwwall** [comment]
I posted a reply on SO at the link that you provided.
